build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
-  id 'maven-publish'
-  id 'io.freefair.lombok' version '6.5.+'
+  id "maven-publish"
+  id "io.freefair.lombok" version "6.5.+"
```

Comment:
We don't have to use maven version syntax for plugins?

Reply:
No. This isn't a published library.